modules/google-maps/src/utils.ts

```diff
@@ -99,17 +108,20 @@ export function getViewPropsFromOverlay(map, overlay) {
   const projection = overlay.getProjection();
 
   const bounds = map.getBounds();
+  assert(bounds);
```

Comment:
Now, this other usages of `assert` may be controversial, so small explanation:

1) `map.getBounds` and similar APIs _according_ to definition can always return `null` or other non-value
2) so, with old code we would have "NullPointerException" here

So, potential bug was lurking, but i wasn't sure how to fix it.

However, typescript complains and we have few ways of addressing this:
 - call `assert` which unfortunately contain _some_ runtime and bundle cos
 - use typescript `!`, but in my opinion, it's ugly and indistigushable from lazy programming
 - spam `// @ts-expect-error` in every usage
 - change code, in a way that we return some kind of no-value (null/undefined) and act accordingly in upper layer like ... skip render in invalid state?
 
I've chosen `assert` as this is fail-safe and most verbose and don't change current behavior, but i understand that we may have other priorities.